proof/invariant-abstract/AARCH64/ArchRetype_AI.thy

```diff
@@ -683,9 +681,7 @@ lemma cap_range_respects_device_region_cong[cong]:
   by (clarsimp simp: cap_range_respects_device_region_def)
 
 
-context begin interpretation Arch .
-requalify_consts region_in_kernel_window
-end
+arch_requalify_consts region_in_kernel_window
```

Comment:
Requalification in an Arch file

Reply:
Unavoidable: const introduced in Arch locale that needs to be referenced generically in this theory.